xiloader/main.cpp

```diff
@@ -193,11 +193,11 @@ int __cdecl main(int argc, char* argv[])
 {
     bool bUseHairpinFix = false;
 
-    /* Output the DarkStar banner.. */
+    /* Output the banner.. */
     xiloader::console::output(xiloader::color::lightred, "==========================================================");
-    xiloader::console::output(xiloader::color::lightgreen, "DarkStar Boot Loader (c) 2015 DarkStar Team");
-    xiloader::console::output(xiloader::color::lightpurple, "Bug Reports: https://github.com/DarkstarProject/darkstar/issues");
-    xiloader::console::output(xiloader::color::lightpurple, "Git Repo   : https://github.com/DarkstarProject/darkstar");
+    xiloader::console::output(xiloader::color::lightgreen, "LandSandBoat Boot Loader (c) 2015 LandSandBoat Team");
```

Comment:
If the repo is changing, we probably should jump forward in time or say 2015-2021

Reply:
That's fair. I'll update this to always show a hyphen with the current year.

It's not strictly required, since the copy-write date of 2015 is unchanged. But It may look nicer.

Reply:
Done. Now shows `2015-<current_year>` e.g. `2015-2021`.

Please take a look and let me know what you think.